R/matching.R

```diff
@@ -294,62 +311,18 @@ plot.maicplus_estimate_weights <- function(x, ggplot = FALSE,
 #' @param processed_agd a data frame, object returned after using \code{\link{process_agd}} or
 #' aggregated data following the same naming convention
 #'
+#' @examples
+#' load(system.file("extdata", "agd.rda", package = "maicplus", mustWork = TRUE))
```

Comment:
@MikeJSeo Do you have the code to generate `agd.rda`? Could you add the code in `data-raw/agd.R`